.github/ISSUE_TEMPLATE/z_bug_report.yml

```diff
@@ -0,0 +1,102 @@
+name: Report an issue with Home Assistant Core (Test)
+about: Report an issue with Home Assistant Core.
+title: ""
+issue_body: true
+inputs:
+  - type: description
+    attributes:
+      value: |
+        This issue form is for reporting bugs only!
+
+        If you have a feature or enhancement request, please use the [feature request][fr] section of our [Community Forum][fr].
+        [fr]: https://community.home-assistant.io/c/feature-requests
+  - type: textarea
+    attributes:
+      label: The problem
+      required: true
+      description: >-
+        Describe the issue you are experiencing here to communicate to the
+        maintainers. Tell us what you were trying to do and what happened.
+
+        Provide a clear and concise description of what the problem is.
+  - type: description
+    attributes:
+      value: |
+        ## Environment
+  - type: input
+    attributes:
+      label: What is version of Home Assistant Core has the issue?
+      required: true
+      placeholder: core-
+      description: >
+        Can be found in the Configuration panel -> Info.
+  - type: input
+    attributes:
+      label: What was the last working version of Home Assistant Core?
+      required: false
+      placeholder: core-
+      description: >
+        If known, otherwise leave blank.
+  - type: dropdown
+    attributes:
+      label: What type of installation are you running?
+      required: true
+      description: >
+        If you don't know, you can find it in: Configuration panel -> Info.
+      choices:
+        - Home Assistant OS
+        - Home Assistant Container
+        - Home Assistant Supervised
+        - Home Assistant Core
+  - type: input
+    attributes:
+      label: Integration causing the issue
+      required: false
+      description: >
+        The name of the integration, for example, Automation or Philips Hue.
+  - type: input
+    attributes:
+      label: Link to integration documentation on our website
+      required: false
+      placeholder: "https://www.home-assistant.io/integrations/..."
+      description: >
+        Providing a link [to the documentation][docs] help us categorizing the
+        issue, while providing a useful reference at the same time.
+
+        [docs]: https://www.home-assistant.io/integrations
+
+  - type: description
+    attributes:
+      value: |
+        # Details
+  - type: textarea
+    attributes:
+      label: Example YAML snippet
+      required: false
+      description: |
+        It this issue has an example piece of YAML that can help reproducing
```

Comment:
If this issue has an example piece of YAML that can help reproduce this problem, please provide that.